### ISSUE TITLE
Dropbox - September 28, 2017: API v1 endpoints are turned off

### ISSUE THREAD

**Issue by chienb** [rclone/rclone#1709]
https://blogs.dropbox.com/developers/2016/06/api-v1-deprecated/

commands getting error:
`2017/09/29 20:18:24 Attempt 1/3 failed with 1 errors and: v1_retired`

**Comment by ncw** [comment]
Please [upgrade to the latest rclone](https://rclone.org/downloads/) - this now uses the v2 API

**Comment by Chinggis6** [comment]
i'm getting the same error for `freebsd 11.1`. installed `rclone` with `pkg` today.

**Comment by detlefs** [comment]
Latest rclone (1.38) still guides me to use v1 of the dropbox API:
```
Please visit:
https://www.dropbox.com/1/oauth2/authorize?client_id=5jcck7diasz0rqy&response_type=code
```
What do I need to do to configure dropbox connection for v2 API?

**Comment by rminderhoud** [comment]
Confirming that upgrading `rclone` version fixes this issue -- dropbox works for me on `1.46`
